installer/datafiles/linux_rpm.data

```diff
@@ -1,10 +1,56 @@
 %Variables
 PERFORMING_UPGRADE_NOT: '[ "$1" -ne 1 ]'
 PACKAGE_TYPE: 'RPM'
+SEPKG_DIR_OMSAGENT: '/usr/share/selinux/packages/omsagent-logrotate'
 
 %Dependencies
 %% In omsadmin.sh: 'curl' requires 'curl',
 curl
 
 omi >= 1.0.8-6
 scx >= 1.6.2-129
+
+%Files
+${{SEPKG_DIR_OMSAGENT}}/omsagent-logrotate.fc;                       installer/selinux/omsagent-logrotate.fc;                                 644; root; root
+${{SEPKG_DIR_OMSAGENT}}/omsagent-logrotate.te;                       installer/selinux/omsagent-logrotate.te;                                 644; root; root
+${{SEPKG_DIR_OMSAGENT}}/omsagent-logrotate.el6.te;                   installer/selinux/omsagent-logrotate.el6.te;                             644; root; root
+${{SEPKG_DIR_OMSAGENT}}/omsagent-logrotate.pp;                       intermediate/${{BUILD_CONFIGURATION}}/selinux/omsagent-logrotate.pp;     755; root; root
+${{SEPKG_DIR_OMSAGENT}}/omsagent-logrotate.el6.pp;                   intermediate/${{BUILD_CONFIGURATION}}/selinux.el6/omsagent-logrotate.pp; 755; root; root
+
+%Directories
+/usr/share/selinux/packages;                                         755; root; root; sysdir
```

Comment:
Looks like if /selinux path does not exist, install fails, check it out: 

cp /home/travis/build/Microsoft/Build-OMS-Agent-for-Linux/omsagent/installer/selinux/omsagent-logrotate.fc /home/travis/build/Microsoft/Build-OMS-Agent-for-Linux/omsagent/intermediate/Linux_UBUNTU_14.04_x64_64_Debug/selinux.el6
cd /home/travis/build/Microsoft/Build-OMS-Agent-for-Linux/omsagent/intermediate/Linux_UBUNTU_14.04_x64_64_Debug/selinux.el6; make -f /usr/share/selinux/devel/Makefile
make[1]: Entering directory `/home/travis/build/Microsoft/Build-OMS-Agent-for-Linux/omsagent/intermediate/Linux_UBUNTU_14.04_x64_64_Debug/selinux.el6'
make[1]: /usr/share/selinux/devel/Makefile: No such file or directory